docs/glossary.md

```diff
@@ -0,0 +1,62 @@
+# OLM Glossary
+
+A list of OLM terms, with definitions and common acronyms.
+
+## CustomResourceDefinitions
+
+### CatalogSources
+
+**Definition**: The CatalogSource is a repository of [CSVs](#ClusterServiceVersion), [CRDs](#CustomResourceDefinitions), and packages that define an application. 
+**Acronyms**: CatSrc(s)
+
+### ClusterServiceVersion
+
+**Definition**: The ClusterServiceVersion is a required resource for an Operator which includes application metadata such as name, description, version, repository link, labels, icon, etc. It also includes a list of corresponding/required resources and their statuses, install strategy of the Operator as [deployments](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/), supported install modes, etc.
+**Acronyms**: CSV(s)
+
+### InstallPlans
+
+**Definition**: The InstallPlan lists out all resources to be created in order to automatically install or upgrade a [CSV](#ClusterServiceVersion). 
+**Acronyms**: IP(s)
+
+### OperatorGroups
+
+**Definition**: The operator group selects a set of target namespaces in which to generate required RBAC access for its member Operators.
+**Acronyms**: OG(s)
+
+### OperatorSources
+
+**Definition**: The operator source defines the external data store used to store Operator bundles. It contains information about the catalog of operators including its name, type, namespace, and display name. 
```

Comment:
`OperatorSources` are external app registries that contains a catalog of operators. Applying an `OperatorSource` to a cluster makes the operators  in that `OperatorSource` available for installation in that cluster. 

For the sentence 
```It contains information about the catalog of operators including its name, type, namespace, and display name. ```
we may want to stick to what these terms mean as a concept, rather than defining *what* the yaml file looks like?